### PR TITLE
`Display` implementations

### PR DESCRIPTION
**Pull request by cafca**
Implement `Display` trait for 

- Author `<Author a5d982>`
- Document `<Document f21e48>`
- DocumentView `<DocumentView 496543_f16e79>`
- Hash `f21e48`
- OperationId `<Operation 6ec805>`
- Schema `<Schema venue 496543_f16e79>` or `<Schema key_group_v1>`

Split out from #268.

## 📋 Checklist

- [x] Add tests that cover your changes
- [x] Add this PR to the _Unreleased_ section in `CHANGELOG.md`
- [x] Link this PR to any issues it closes
- [x] New files contain a SPDX license header
- [x] Check if descriptions and terminology match `handbook` content (and visa-versa)

### CODE REVIEW

File: p2panda-rs/src/document/document_view_id.rs [hunk 1]
```diff
@@ -63,6 +64,17 @@ impl DocumentViewId {
     }
 }
 
+impl Display for DocumentViewId {
+    /// Document view ids are displayed by concatenating their hashes with an underscore separator.
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        for (i, operation_id) in self.0.clone().into_iter().enumerate() {
+            let separator = if i == 0 { "" } else { "_" };
```
Comment (sandreae) [inline p2panda-rs/src/document/document_view_id.rs:71]: Check out `peek()` which helps with this kind of pattern : https://doc.rust-lang.org/std/iter/struct.Peekable.html

Not strictly necessary here cos it's already very neat, but just for interest.

**Review by sandreae — APPROVED** [review]
Morning! All looks good to me, thanks for splitting all these PRs up, really easy to review now :+1:

In answer to your top question, I'd be in favour of using a some shorthand version in the display implementation. I'm also happy with this for now, and maybe we understand better what kind of shorthand is useful when we start debugging.

File: p2panda-rs/src/document/document_view_id.rs (hunk 1 above)
Comment (cafca) [reply p2panda-rs/src/document/document_view_id.rs:71]: I saw that you used that yesterday hehe, also quite interesting.

**Comment by cafca** [comment]
I'll update this to make short representations

**Comment by cafca** [comment]
I still want to go forward with this but it's a bit on the back burner.

**Review by adzialocha — APPROVED** [review]

**Pull request merged**